deps/uv/src/win/internal.h

```diff
@@ -332,8 +332,8 @@ int WSAAPI uv_wsarecvfrom_workaround(SOCKET socket, WSABUF* buffers,
     int* addr_len, WSAOVERLAPPED *overlapped,
     LPWSAOVERLAPPED_COMPLETION_ROUTINE completion_routine);
 
-int WSAAPI uv_msafd_poll(SOCKET socket, AFD_POLL_INFO* info,
-    OVERLAPPED* overlapped);
+int WSAAPI uv_msafd_poll(SOCKET socket, AFD_POLL_INFO* info_in,
```

Comment:
The file name `internal.h` suggests that it's an internal API, but the function name (`uv_msafd_poll` and not `uv__msafd_poll`) seems to suggest that it's part of the public API. Is it a change to the public API?